README.md

```diff
@@ -0,0 +1,103 @@
+# Hazelcast Open Binary Client Protocol
+
+**Hazelcast Open Binary Client Protocol** definitions and code generator for multiple programming languages.
+
+## Hazelcast Open Binary Client Protocol Definitions
+
+The protocol is defined in `protocol-definitions/*.yaml` yaml files where each yaml file represents a service like Map, List, Set etc.
+
+## Service definition
+
+A service is defined by a separate YAML file, containing all its method definitions.
+
+```yaml
+id: Service Id (1-255)
+name: Service Name
+methods:
+  - id: METHOD-ID-1 (1-255)
+    name: METHOD-NAME-1
+    ...
+  - id: METHOD-ID-2 (1-255)
+    name: METHOD-NAME-2
+    ...
+```
+
+A method(aka Remote method call) is defined by a request-response pair.
+
+A basic method structure example:
+
+```yaml
+  - id: METHOD-ID-1 (1-255)
```

Comment:
The old protocol README (current master) explains what each property means. I think we should do the same for the new protocol too. (like what `since`, `retryable`, `acquires-resource` ... means)

Reply:
I think we can use schema doc for that purpose and just refer to it.